src/_includes/docs/swift-package-manager/migrate-ios-project-manually.md

```diff
@@ -0,0 +1,84 @@
+Once you [turn on Swift Package Manager][] support, the Flutter CLI tries to
+migrate your project to add Swift Package Manager integration.
+This makes your project download the Swift packages that your Flutter
+plugins depend on.
+
+However, the Flutter CLI might be unable to migrate your project if you've
```

Comment:
Do we mean "even if you've modified it significantly" ?

Note: "even" here?

Reply:
We can migrate projects _unless_ they've received heavy modifications.

We match do "match and replace" in your Xcode project to migrate it. This should work fine if your project is based off Flutter's templates. However, our matching logic can break if you've modified your Xcode project. The bigger your modifications, the more likely our matching logic breaks.

Reply:
I guess I'm wondering if we need the "significantly" qualifier?

ideas:

"The Flutter CLI attempts to migrate your project to Swift Package Manager, even if you have made changes to <foo>. If the Flutter CLI tool is unable to migrate your project (for example, if there are unexpected modifications to Foo or Bar), then the Flutter CLI tool will <print some error or warning telling you where to manually update and modify>."

Reply:
Ah gotcha, I dropped the "significantly" qualifier and switched to wording similar to what you suggested.